submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/submission/submit/ResubmissionApiTest.kt

```diff
@@ -180,4 +187,65 @@ class ResubmissionApiTest(
 
         assertThat(webClient.submitSingle(submission, TSV)).isSuccessful()
     }
+
+    @Test
+    fun `5-4 regular user tries to resubmit (un-release) with future release date`() {
+        val version1 = tsv {
+            line("Submission", "S-RSTST4")
+            line("Title", "Public Submission")
+            line("ReleaseDate", OffsetDateTime.now().toStringDate())
+            line()
+            line("Study")
+            line()
+        }.toString()
+
+        val regularUser = RegularUser.email
+        val onBehalfClient = SecurityWebClient
```

Comment:
is not clear why we mix the onBehalf functionally here, I think we should simple use a regular use client.